docs/user-guide/faq.md

```diff
@@ -186,14 +188,24 @@ If you need to reference an old version of a dataset, you can find previous vers
 Data from AACR Project GENIE are provided in a [dedicated instance of cBioPortal](https://www.cbioportal.org/genie/). You can also download GENIE data from the [Synapse Platform](https://synapse.org/genie). Note that you will need to register before accessing the data. Additional information about AACR Project GENIE can be found on the [AACR website](https://www.aacr.org/Research/Research/Pages/aacr-project-genie.aspx).
 
 ### TCGA
-#### How does TCGA data in cBioPortal compare to TCGA data in Genome Data Commons?
-We do not currently load the mutation data from the GDC. Instead, we have the original mutation data generated by the individual TCGA sequencing centers. The source of the data is the Broad Firehose (or the publication pages for data that matches a specific manuscript). These data are usually a combination of two mutation callers, but they differ by center (typically a variant caller like MuTect plus an indel caller), and sequencing centers have modified their mutation calling pipelines over time.
+#### What are the TCGA studies sourced from the Genomic Data Commons (GDC)?
+The GDC TCGA studies mirror the [Cancer Gateway in the Cloud (ISB-CGC)](https://portal.isb-cgc.org) that is hosted on Google BigQuery, which in turn pulls data from GDC. The [NCI-CRDC pipeline](https://github.com/cBioPortal/nci-crdc-pipeline) pulls data from ISB-CGC and transforms it into cBioPortal-formatted files. The resulting studies are intended to be a pure reflection of what is available inside ISB-CGC; we do not augment them with data from our legacy studies.
+
+For more information on how GDC data is processed, refer to the following links:
+- [GDC Data Processing](https://gdc.cancer.gov/about-data/gdc-data-processing)
+- [ISB-CGC Programs and Data Sets](https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/Hosted-Data.html)
+- [ISB-CGC GDC Overview](https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/GDC_top.html)
+- [NCI-CRDC Pipeline Data Transformations](https://github.com/cBioPortal/datahub/blob/gdcv1/crdc/gdc/brca_tcga_gdc/README.md) (may vary by study)
+#### How do the GDC TCGA studies compare to the TCGA PanCancer / Firehose Legacy studies?
```

Comment:
I would use the same label here:

"How do the TCGA Studies sourced from GDC compare to the TCGA PanCancer  and Firehose Legacy studies?"